CHANGELOG.md

```diff
@@ -1,3 +1,3 @@
-## 1.3.0
+## 1.3.1
```

Comment:
Changelog should have changes happened in all the versions with latest version on top.
Don't remove the older ones

https://github.com/Alamofire/Alamofire/blob/master/CHANGELOG.md

Reply:
sure